readme-vars.yml

```diff
@@ -53,36 +53,34 @@ app_setup_block: |
 
   Emby has very complete and verbose documentation located [here](https://github.com/MediaBrowser/Wiki/wiki) .
 
-  Hardware acceleration users for Intel Quicksync and AMD VAAPI will need to mount their /dev/dri video device inside of the container by passing the following command when running or creating the container:
+  ### Hardware Acceleration Enhancements
 
-  ```--device=/dev/dri:/dev/dri```
+  This list out the enhancements we have explicit made for hardware acceleration in this image.
```

Comment:
Perhaps modify as `This section lists the enhancements we have made for hardware acceleration in this image specifically.`